src/Tinfoil/Key.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Tinfoil.Key(
+    genSymmetricKey256
+) where
+
+import           P
+
+import           System.IO (IO)
+
+import           Tinfoil.Data.Key
+import           Tinfoil.Data.Random
+import           Tinfoil.Random
+
+-- | Generate a 256-bit symmetric cryptographic key.
+genSymmetricKey256 :: IO SymmetricKey
+genSymmetricKey256 = fmap (SymmetricKey . unEntropy) $ entropy 32
```

Comment:
Ah yeah, I used `hlint` until I discovered how much we disagreed about good style. I don't really have a preference, I just default to `fmap` most of the time because `<$>` sometimes gets unwieldy in larger constructions and I prefer to be consistent. Not something I care about, so if anyone has a strong preference either way, speak up.